tasties_app/templates/tasties_app/view_recipe.html

```diff
@@ -50,5 +50,28 @@ <h3>Directions</h3>
             <p>{{ recipe.directions }}</p>
         </div>
         <hr>
+        <section class="comment-section">
+            <div id="comment-group" class="form-group">
+                <h3 for="exampleFormControlTextarea1">Comments</h3>
+                {% for comment in comments %}
+                    <div class="comment-container">
+                        <div class="container">
+                            <div class="row">
+                                <div class="col col-lg-2" id="author-name">By @{{ comment.author_id }}<br>{{ comment.publication_date }}</div>
+                                <div class="col-sm" id="comments-text">{{ comment.comment_text }}</div>
+                            </div>
+                        </div>
+                    </div>
+                {% endfor %}
+                <form method="POST" >
```

Comment:
Maybe you should give the post method `name` and `value` attributes,
Because when we will have another functionality i.e. adding a rating, we would like to know when the `POST` method refers to adding a comment, and when it refers to doing something else.
[Here ](https://stackoverflow.com/questions/18415161/differentiating-between-different-post-requests-on-the-same-page-in-django-views)is an example of doing it right.

Note that you should also change it on the views method.